aioconsole/rlwrap.py

```diff
@@ -83,6 +84,9 @@ def write(arg):
             dest.write(arg)
             dest.flush()
 
+    # Prevent BlockingIOError on macOS.
+    fcntl.fcntl(dest.fileno(), fcntl.F_SETFL, 0)
```

Comment:
If it's a macOS bug, should we make it a conditional statement? Any reason for this line to cause other bugs on linux? Can it raise an exception?

Reply:
> If it's a macOS bug, should we make it a conditional statement?

Good point. I've updated it to check the platform.

>  Any reason for this line to cause other bugs on linux? 

Considering how badly it goes when the file descriptor becomes blocking I would be surprised, but better to make a narrow change.

> Can it raise an exception?

Only if something goes terribly wrong (like the file descriptor is closed), in which case you'd probably prefer to get an exception:

https://github.com/python/cpython/blob/master/Modules/fcntlmodule.c#L59

> Also please let me know if you need a new release :)

Yes, please. It's no fun living on a fork.